velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1399,4 +1405,42 @@ struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
   }
 };
 
+template <typename T>
+struct ToISO8601FromTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  std::string sessionTzName = "";
+  std::string tzOffsetStr = "";
```

Comment:
What is the purpose of having this as a member?

Reply:
No purpose - moved declaration to `call()`.